buildpacks/jvm-function-invoker/src/main.rs

```diff
@@ -87,7 +87,7 @@ impl Buildpack for JvmFunctionInvokerBuildpack {
                     .process(
                         ProcessBuilder::new(
                             process_type!("web"),
-                            layers::opt::JVM_RUNTIME_SCRIPT_NAME,
+                            ["bash", layers::opt::JVM_RUNTIME_SCRIPT_NAME],
```

Comment:
Something I'm going to explore next week for the procfile-cnb PR, is what happens when additional args are passed in by the user at runtime. In the new "direct=true with explicit bash" world, I wonder if the commands would get interpreted correctly, or whether they would end up as ~`bash -c "<command>" --args` with bash not treating `--args` as part of the command string. (I think with direct=false mode there was more magic happening upstream that might have helped that case.)

Reply:
If I understood you correctly, this isn't a blocker for this PR though, correct? :) I would love to merge this sooner rather than later since I have changes lined up that build on top of this PR.

Reply:
Separate to the args point above, switching to `direct=true` means `.profile.d/` scripts and the `.profile` script will no longer be sourced prior to the `JVM_RUNTIME_SCRIPT_NAME` script being run. This probably doesn't matter much for functions, where most users won't be using custom legacy buildpacks or know about the `.profile` feature - but calling it out just in case :-)